bundles/org.openhab.binding.fronius/README.md

```diff
@@ -64,26 +64,37 @@ The binding has no configuration options, all configuration is done at `bridge`,
-| `powerfactorphase3`     | Number    | Power Factor on Phase 3                                                                                                                                                                                                  |
-| `energyrealsumconsumed` | Number    | Real Energy consumed                                                                                                                                                                                                     |
-| `energyrealsumproduced` | Number    | Real Energy produced                                                                                                                                                                                                     |
+| Channel ID              | Item Type              | Description                                                                                                                                                                                                              |
+| ----------------------- | ---------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `enable`                | Number                 | 1 = enabled, 0 = disabled                                                                                                                                                                                                |
+| `location`              | Number                 | 0 = grid interconnection point (primary meter)<br/> 1 = load (primary meter)   <br />3 = external generator (secondary meters)(multiple)<br />256-511 = subloads (secondary meters)(unique). Refer to Fronius Solar API. |
+| `currentacphase1`       | Number:ElectricCurrent | AC Current on Phase 1                                                                                                                                                                                                    |
+| `currentacphase2`       | Number:ElectricCurrent | AC Current on Phase 2                                                                                                                                                                                                    |
+| `currentacphase3`       | Number:ElectricCurrent | AC Current on Phase 3                                                                                                                                                                                                    |
+| `voltageacphase1`       | Number:Voltage         | AC Voltage on Phase 1                                                                                                                                                                                                    |
```

Comment:
This must be `Number:ElectricPotential`.

Reply:
fixed